### PR TITLE
Fix command line docs and update complete-guide

### PR DESCRIPTION
**Pull request by harshavardhana**
Also re-ordered snap below binary download for windows in priority.

### CODE REVIEW

File: cmd/session-clear.go [hunk 1]
```diff
@@ -44,16 +44,20 @@ var sessionClear = cli.Command{
   
 USAGE:
   {{.HelpName}} SESSION-ID|all
+
+SESSION-ID:
+  SESSION - Session can either be $SESSION-ID
```
Comment (krisis) [inline cmd/session-clear.go]: this seems incomplete. Shouldn't this be `$SESSION-ID` or `all`?

File: cmd/session-resume.go [hunk 2]
```diff
@@ -38,14 +38,17 @@ var sessionResume = cli.Command{
   {{.HelpName}} - {{.Usage}}
 
 USAGE:
-  {{.HelpName}} [SESSION-ID/all]
+  {{.HelpName}} SESSION-ID
+
+SESSION-ID:
```
Comment (krisis) [inline cmd/session-resume.go]: this seems incomplete. Shouldn't this be `$SESSION-ID` or `all`?

File: docs/minio-client-complete-guide.md [hunk 3]
```diff
@@ -23,23 +23,64 @@ version  Print version info.
...
+
+```sh
+chmod +x mc
+./mc --help
+```
+
+### Binary Download (Microsoft Windows)
+| Platform | Architecture | URL |
+| ---------- | -------- |------|
+|Microsoft Windows|64-bit|https://dl.minio.io/client/mc/release/windows-amd64/mc.exe |
```
Comment (krisis) [inline docs/minio-client-complete-guide.md]: why are we calling this 64-bit and not 64-bit Intel?

File: cmd/session-resume.go [hunk 4]
```diff
@@ -38,14 +38,17 @@ var sessionResume = cli.Command{
...
+  SESSION - Session can either be $SESSION-ID
```
Comment (poornas) [inline cmd/session-resume.go]: incomplete like @krisis pointed in previous instances

File: cmd/session-clear.go (hunk 1 above)
Comment (harshavardhana) [reply cmd/session-clear.go]: yes it should be `all`

File: cmd/session-resume.go (hunk 2 above)
Comment (harshavardhana) [reply cmd/session-resume.go]: No resume is not all @krisis

File: docs/minio-client-complete-guide.md (hunk 3 above)
Comment (harshavardhana) [reply docs/minio-client-complete-guide.md]: Old baggage..

File: cmd/session-resume.go (hunk 4 above)
Comment (harshavardhana) [reply cmd/session-resume.go]: Not incomplete `resume` is purposefully not supporting `all`

Comment (krisis) [reply cmd/session-resume.go]: @harshavardhana then the description shouldn't be `SESSION - Session can either be ...`, right?

Comment (harshavardhana) [reply cmd/session-resume.go]: yes removing @krisis

**Comment by harshavardhana** [comment]
can this be reviewed again @krisis @poornas ?

**Pull request merged**